src/revwalk.rs

```diff
@@ -21,6 +34,7 @@ impl<'repo> Revwalk<'repo> {
         unsafe {
             try_call!(raw::git_revwalk_reset(self.raw()));
         }
+        self.hide_cb = None;
```

Comment:
To confirm, `git_revwalk_reset` clears the callback internally in libgit2, right? So this is just reflecting how the Rust callback will never get called anyway?

Reply:
Yes that's right. I'll add a comment.